test/e2e-common.sh

```diff
@@ -98,7 +98,9 @@ function test_task_creation() {
         local runtestdir=${runtest#*/}
         # remove /0.1/tests from end
         local testname=${runtestdir%%/*}
-        local tns="${testname}-$$"
+        # get version of the task and replace . with - in version as not supported in namespace name
+        local version="$( echo $(basename $(basename $(dirname $runtest))) | tr '.' '-' )"
```

Comment:
looking good, can we have a little check if version is defined ? just in case if the task contributor hasn't conformed properly to the naming system ?

Reply:
Something like this : 
```suggestion
        local version="$( echo $(basename $(basename $(dirname $runtest))) | tr '.' '-' )"
	    [[ ${version} =~ ^[0-9]+\.[0-9]+$ ]] || { echo "ERROR: version of the task is not set properly"; exit 1;}
```

Reply:
Addressed